apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStoreParallelLiveTest.java

```diff
@@ -105,13 +109,40 @@ public void setup() throws IOException, InterruptedException {
    public void cleanupFiles() {
       // Delete local file
       delete(bigFile);
+      delete(smallFile);
       delete(new File(bigFile + ".downloaded"));
 
       // Delete uploaded file
       blobStore.clearContainer(CONTAINER);
       blobStore.deleteContainer(CONTAINER);
+      blobStore.clearContainer(CONTAINER_TEMP);
+      blobStore.deleteContainer(CONTAINER_TEMP);
    }
 
+   @Test(expectedExceptions = { IllegalStateException.class })
+   public void getBlobTestFailure() {
+      blobStore.containerExists(CONTAINER_TEMP);
```

Comment:
Sorry you still have not explained what this pull request does.  What does this test demonstrate and why is it in the parallel class instead of `RegionScopedBlobStoreContextLiveTest`?